### PR TITLE
#26 - update to compatibility with laravel 11

### PR DESCRIPTION
**Pull request by JakubKermes**
Bumped dependencies. Based on Laravel [documentation](https://laravel.com/docs/11.x/upgrade) for upgrading Laravel, there is no need for changing any code.

### CODE REVIEW

File: composer.json
Comment (orlowski11) [inline composer.json:1]: There is some kind of problem with this, check slack for more information.

Comment (JakubKermes) [reply composer.json:1]: Hey, are you sure this is about this branch?

**Review by orlowski11 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blumilksoftware/blt",
+  "name":"blumilksoftware/blt",
```
Comment (krzysztofrewak) [inline composer.json:2]: Why did you removed a space here?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (JakubKermes) [reply composer.json]: Sorry, fixed

File: composer.json [hunk 2]
```diff
@@ -10,18 +10,18 @@
...
-    "blumilksoftware/codestyle": "^1.10",
-    "illuminate/contracts": "^9.1|^10.0",
-    "illuminate/http": "^9.1|^10.0",
-    "illuminate/support": "^9.1|^10.0",
-    "symfony/http-foundation": "^6.0"
+    "blumilksoftware/codestyle": "^3.1.0",
+    "illuminate/contracts": "^10.0|11.9.2",
+    "illuminate/http": "^10.0|11.9.2",
+    "illuminate/support": "^10.0|11.9.2",
+    "symfony/http-foundation": "^6.2|^7.1"
```
Comment (krzysztofrewak) [inline composer.json:24]: I think you can stay with `"^10.0|11.9"` or even `"^10.0|11.0"` if there's no big changes in 11.1.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

**Review by PiotrFedak — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**